src/main/java/org/opensearch/flowframework/util/EncryptorUtils.java

```diff
@@ -271,9 +279,20 @@ public void initializeMasterKey(ActionListener<Boolean> listener) {
 
                 } else {
                     // Set existing key to master
-                    logger.info("Config has already been initialized");
-                    this.masterKey = (String) getResponse.getSourceAsMap().get(MASTER_KEY);
-                    listener.onResponse(true);
+                    logger.debug("Config has already been initialized, fetching key");
```

Comment:
Do we need to mention anything about the key here?
```suggestion
                    logger.debug("Config has already been initialized");
```